src/components/organisms/SchedulePageModal/utils.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  differenceInMinutes,
+  endOfDay,
+  getUnixTime,
+  isWithinInterval,
+  max,
+  min,
+  startOfDay,
+} from "date-fns";
+
+import { Room } from "types/rooms";
+import { PersonalizedVenueEvent, VenueEvent } from "types/venues";
+import { MyPersonalizedSchedule } from "types/User";
+
+import { WithVenueId } from "utils/id";
+import { eventEndTime, eventStartTime } from "utils/event";
+import { isTruthy } from "utils/types";
+
+import { RoomWithEvents } from "components/molecules/Schedule/Schedule.types";
```

Comment:
This type should be moved into `types/rooms` or similar.

Reply:
don't need RoomWithEvents anymore: https://github.com/sparkletown/sparkle/pull/1302/commits/e3803174b3455512104c73a78f74d8fba8962b1c